config/owasp/suppressions.xml

```diff
@@ -2234,6 +2234,76 @@
     </suppress>
     <suppress>
         <notes><![CDATA[
+   file name: htmlunit-driver-2.36.0.jar
```

Comment:
Remove these CVEs, check Abhijit's email for this.

Reply:
They have been removed in this commit: https://github.com/hmcts/rd-user-profile-api/pull/249/commits/4774a1cdadb7db0645df942a7dcd38a781f9f52a